# FootOutputMsg_v2.msg

time stamp
int16 id

float32[5] joint_position

float32[5] joint_speed

float32[5] desired_efforts

float32[5] actual_efforts

int16 state
